src/js/popup.ts

```diff
@@ -178,8 +185,11 @@ function attachListeners(origin: string | null): void {
 }
 
 function updateDisplay(state: string | null): void {
-  const popupState = STATE_TO_POPUP_STATE[state] || state;
+  const popupState = STATE_TO_POPUP_STATE[state!] || state;
   Array.from(document.getElementsByClassName('state_boundary')).forEach(
+    // @ts-ignore: getElementsByClassName returns HTMLCollectionOf<Element>
```

Comment:
`getElementsByClassName` returns all `Elements` matching the classname, could be `HTMLElement`s or `SVGElement`s or whatever is extending [`Element`](https://developer.mozilla.org/en-US/docs/Web/API/Element). Can we adjust the check below to avoid this ignore?

Reply:
Yes! Adding an `instanceof HTMLElement` check fixes it.